extensions/wikia/Scribe/ScribeEventProducer.class.php

```diff
@@ -209,14 +238,9 @@ public function buildUndeletePackage( $oTitle, $created = false ) {
 			return true;
 		}
 
-		$oLocalFile = null;
-		if ( $created && $oTitle->getNamespace() == NS_FILE ) {
-			$oLocalFile = wfLocalFile( $oTitle );
-		}
-
 		wfProfileOut( __METHOD__ );
 
-		return $this->buildEditPackage( $oPage, $oUser, null, $oLocalFile );
+		return $this->buildEditPackage( $oPage, $oUser );
 	}
 
 	public function buildMovePackage( $oTitle, $oUser, $page_id = null, $redirect_id = null ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)